docker/fundamentals/images/dockerfile-ii.md

```diff
@@ -0,0 +1,62 @@
+---
+author: nem035
+aspects:
+  - introduction
+  - workout
+type: normal
+category: must-know
+links:
+- '[Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/){article}'
+- '[Dockerfile reference](https://docs.docker.com/engine/reference/builder/){documentation}'
+---
+
+# Dockerfile (Part 2)

```

Comment:
Let's name this one 

```suggestion
# Dockerfile Syntax

```

Reply:
And also change the filename (slug)

Reply:
Hmmm actually the first one is probably better as "Dockerfile Syntax" haha

Forget I said anything, it's good as is for now

Reply:
how about:
1. Dockerfile Syntax
2. Dockerfile Instructions

Reply:
Let's do

1. Dockerfile Format
2. Dockerfile Instructions